src/deep-security/lib/CredentialsManager.js

```diff
@@ -147,10 +147,10 @@ export class CredentialsManager {
           return callback(true);
         });
       },
-      onDatasetDeleted: (dataset, datasetName, callback) => {
+      onDatasetDeleted: (dataset, datasetName) => {
         return callback(true);
       },
-      onDatasetMerged: (dataset, datasetNames, callback) => {
+      onDatasetMerged: (dataset, datasetNames) => {
```

Comment:
'datasetNames' is defined but never used.<br>'dataset' is defined but never used.